pkg/provider/azure_loadbalancer.go

```diff
@@ -1436,6 +1529,11 @@ func (az *Cloud) reconcileLoadBalancer(clusterName string, service *v1.Service,
 		return nil, fmt.Errorf("reconcileLoadBalancer: failed to list managed LB: %w", err)
 	}
 
+	if err := az.reconcileMultipleStandardLoadBalancerConfigurations(clusterName, &existingLBs); err != nil {
+		klog.Errorf("reconcileLoadBalancer: invalid multiple standard load balancers configurations: %s", err.Error())
+		return nil, err
```

Comment:
I put the comment because I saw the following change in this PR and I think they should be of the same code style, right?
```
	svcs, err := az.KubeClient.CoreV1().Services("").List(context.Background(), metav1.ListOptions{})
	if err != nil {
		klog.Errorf("reconcileMultipleStandardLoadBalancerConfigurations: failed to list all load balancer services: %w", err)
		return fmt.Errorf("failed to list all load balancer services: %w", err)
	}
```

Reply:
The original error here is returned by List and the message is not under my controller so I enrich the error message.